mirrord/agent/src/sniffer.rs

```diff
@@ -287,6 +292,8 @@ where
             } => {
                 if self.port_subscriptions.unsubscribe(client_id, port) {
                     self.update_packet_filter()?;
+
+                    MIRROR_PORT_SUBSCRIPTION.dec();
```

Comment:
Now that I think of it, looks like you also need to decrement in `handle_client_closed`